File: Ejercicio1.cpp
//Ejercicio 1: Sumar dos números y presentar el resultado.
#include <iostream>
#include <chrono>

using namespace std;
using namespace std::chrono;

int main() {
    cout << "--- Ejercicio 1: Sumar dos numeros ---\n";

    // Inicia cronómetro
    auto start = high_resolution_clock::now();

    int a = 10;
    int b = 20;
    int suma = a + b;
    cout << "La suma es: " << suma << "\n";

    // Detiene cronómetro
    auto end = high_resolution_clock::now();
    duration<double> time_span = duration_cast<duration<double>>(end - start);
    
    cout << "Tiempo de ejecucion: " << time_span.count() << " segundos\n\n";
    return 0;
}

File: Ejercicio1.py
#Ejercicio 1: Sumar dos números y presentar el resultado.
import time

print("--- Ejercicio 1: Sumar dos números ---")

# Inicia cronómetro
start_time = time.time()

a = 10
b = 20
suma = a + b
print("La suma es:", suma)

# Detiene cronómetro
end_time = time.time()
print(f"Tiempo de ejecución: {end_time - start_time:.6f} segundos\n")